tests/common/auth_util.go

```diff
@@ -24,72 +24,72 @@ import (
 )
 
 const (
-	rootUserName = "root"
-	rootRoleName = "root"
-	rootPassword = "rootPassword"
-	testUserName = "test-user"
-	testRoleName = "test-role"
-	testPassword = "pass"
+	RootUserName = "root"
+	RootRoleName = "root"
+	RootPassword = "rootPassword"
+	TestUserName = "test-user"
+	TestRoleName = "test-role"
+	TestPassword = "pass"
 )
 
 var (
-	rootUser = authUser{user: rootUserName, pass: rootPassword, role: rootRoleName}
-	testUser = authUser{user: testUserName, pass: testPassword, role: testRoleName}
-
-	testRole = authRole{
-		role:       testRoleName,
-		permission: clientv3.PermissionType(clientv3.PermReadWrite),
-		key:        "foo",
-		keyEnd:     "",
+	RootUser = AuthUser{User: RootUserName, Pass: RootPassword, Role: RootRoleName}
+	TestUser = AuthUser{User: TestUserName, Pass: TestPassword, Role: TestRoleName}
+
+	TestRole = AuthRole{
+		Role:       TestRoleName,
+		Permission: clientv3.PermissionType(clientv3.PermReadWrite),
+		Key:        "foo",
+		KeyEnd:     "",
 	}
 )
 
-type authRole struct {
-	role       string
-	permission clientv3.PermissionType
-	key        string
-	keyEnd     string
+type AuthRole struct {
+	Role       string
+	Permission clientv3.PermissionType
+	Key        string
+	KeyEnd     string
```

Comment:
Exporting each field isn't a best practice to me. I would suggest to add a method something like below for each field,

```
func (ar AuthRole) Role() string {
    return ar.role
}
```

Since currently all fields will not written from outside the package, so no need to add method something like `SetRole(string)` for now.

Reply:
I don't have anything against this. Getters and setters are overkill here.